src/Entity.php

```diff
@@ -192,7 +192,9 @@ public function fill($data)
                 }
                 else
                 {
-                    $value = static::buildEntity($value);
+                    $stdCls = new StdClass();
```

Comment:
@ankitdas13 , why the fix going outside? can't we place it `static::buildEntity`
Also looks like we are missing tests here.

Reply:
@ramth05 We are only replacing Entity (Payment,Order,Subscriptions etc..) instance to `StdObject` for associative array and `static:buildEntity` was there before but while fixing this [bug](https://github.com/razorpay/razorpay-php/issues/15) (raised by nemo) i only got this solution if you have any approach then please suggest me.